modules/button/react/stories/visual-testing/stories_ToolbarIconButton.tsx

```diff
@@ -0,0 +1,43 @@
+/// <reference path="../../../../../typings.d.ts" />
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import * as React from 'react';
+import {StaticStates} from '@workday/canvas-kit-labs-react-core';
+import {
+  ComponentStatesTable,
+  withSnapshotsEnabled,
+  permutateProps,
+} from '../../../../../utils/storybook';
+import {playCircleIcon} from '@workday/canvas-system-icons-web';
+import {ToolbarIconButton, IconButton} from '../../index';
+import {Container, stateTableColumnProps} from './utils';
+
+export default withSnapshotsEnabled({
+  title: 'Testing|React/Buttons/Button/Toolbar Icon Button',
+  component: ToolbarIconButton,
+});
+
+export const ToolbarIconButtonStates = () => (
+  <React.Fragment>
+    {[false, true].map(toggled => (
```

Comment:
Does toggled really need to be another table? We did it for icon buttons so it was a little easier to read, but when there is only one row per table, why not just combine them and make `toggled` a row prop?